src/main/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/messages.properties

```diff
@@ -2,7 +2,7 @@ containsTab=Line contains a tab character.
 empty.line.separator=''{0}'' should be separated from previous statement.
 empty.line.separator.multiple.lines=''{0}'' has more than 1 empty lines before.
 empty.line.separator.multiple.lines.after=''{0}'' has more than 1 empty lines after.
-empty.line.separator.multiple.lines.inside=There is more than 1 empty line one after another.
+empty.line.separator.multiple.lines.inside=Line ''{0}'' has more than 1 empty line one after another.
```

Comment:
this is point of confusion ....

Violation should be reported on last known printable token in code.
Message should be `There is more than 1 empty line after this line.`
we should not point at line that we think is not good, we absolutely know last line with visible symbols and fact that there more then 1 line after it.

Reply:
Done.